src/api/time-series.js

```diff
@@ -0,0 +1,61 @@
+const { request } = require('./graphql')
+
+const TIME_SERIES_QUERY = `
+  query TimeSeries(
+    $site: String!
+    $measurements: [MetricTag!]
+    $pages: [String!]
+    $from: String,
+    $to: String
+  ) {
+    organisation {
+      timeSeries(site: $site, from: $from, to: $to, measurements: $measurements, pages: $pages) {
+        times {
+          name
```

Comment:
Times?

Reply:
Yeah, timeSeries returns all the `times` and then the different `series` which match those times.